src/components/tasks/task-dialog.tsx

```diff
@@ -353,14 +457,46 @@ const TaskDialog = ({ projectId, milestoneId, open, task, onClose }: Props) => {
               <TableRow>
                 <TableCell>{t("newMilestoneTaskDialog.taskAttachmentsTable.type")}</TableCell>
                 <TableCell>{t("newMilestoneTaskDialog.taskAttachmentsTable.name")}</TableCell>
-                <TableCell>{t("newMilestoneTaskDialog.taskAttachmentsTable.size")}</TableCell>
+                <TableCell>{t("newMilestoneTaskDialog.taskAttachmentsTable.preview")}</TableCell>
+                <TableCell>{t("newMilestoneTaskDialog.taskAttachmentsTable.delete")}</TableCell>
               </TableRow>
             </TableHead>
             <TableBody>
-              <TableRow>
-                <TableCell>Attachment name</TableCell>
-                <TableCell>Size</TableCell>
-              </TableRow>
+              {taskData.attachmentUrls.map((attachment, index) => (
+                <TableRow key={index}>
+                  <TableCell>{getAttachmentTypeFromUrlCapitals(attachment)}</TableCell>
+                  <TableCell>{attachment}</TableCell>
+                  <TableCell>
+                    <Button variant="text" color="primary" sx={{ borderRadius: 25 }} onClick={() => {
```

Comment:
This indentation is slightly off, do you not have biome working so that it automatically fixes indentation?

Reply:
Used it now